tests/indent/python/return_dedent.py

```diff
@@ -14,3 +14,26 @@ def a():
     return (
         1, 2, 3
     )
+
+def a():
+    return b(
+        1, 2, 3
```

Comment:
how would 
```
b(1,
2,
3)
````
be indented?

Reply:
Interesting, pressing enter on that line leads to
```python
b(1,
2,
3)
  |
```

Reply:
I think @georgeharker said that he's planning to change the behavior to support this kind of style as well, so I am not sure whether I should try to implement it here

Reply:
No, I meant what would happen when you indent the whole expression or when you start a new line after
````
b(1,
```
?